src/store/index.js

```diff
@@ -8,7 +8,7 @@ export default new Vuex.Store({
   state: {
     moviesRecord: [],
     genreList: [],
-    selectedGenre: "",
+    selectedGenre: "All Categories",
```

Comment:
I'm actually not sure if this is the best solution 🤔  I see another option, what do you say for changing a bit how a select "placeholder" works? You can add `value` attribute to `<option>` tha will be set if this option is seleced - value does not need to be the same as label e.g.

```html
<option value="">All Categories</option>
```
this way empty string will be the value of `<select>` if this option is selected

what do you think?

Reply:
I can see where you are coming from with the value attribute 👍🏽  I have updated it as you suggested hope this is ok. & THANK YOU! :)